### PR TITLE
chore: bump to v0.53.0

### PR DESCRIPTION
**Pull request by JoshuaBatty**
## Description
Bump repo to 0.53.0

waiting on #5813

### CODE REVIEW

**Review by kayagokalp — CHANGES_REQUESTED** [review]
Requesting change to block this from getting merged accidentally as #5824 got merged and this will need to become 0.53

**Review by kayagokalp — APPROVED** [review]
approving to unblock

**Review by IGI-111 — APPROVED** [review]

**Pull request merged**
